.github/workflows/pull-requests.yml

```diff
@@ -155,7 +155,7 @@ jobs:
         id: fc-initial
         with:
           issue-number: ${{ github.event.pull_request.number }}
-          comment-author: 'github-actions[bot]'
+          comment-author: "github-actions[bot]"
```

Comment:
It looks like for some reason this file was formatted with different prettier settings, or maybe without prettier? This change will be reverted by the next person to save a change in this file.

Reply:
mmh we have a mixture of single and double quotes in that file tho